docs/user_guide/emummc/partitioning_sd.md

```diff
@@ -3,77 +3,48 @@
-4. Create a new partition by right clicking on the unallocated space and then pressing `Create` with the following instructions:
-	- Set the metrics on the right of `Unallocated Space Before` and `Unallocated Space After` to `MB` if it's set to `GB`.
-	- Set the metrics on the right of `Partition Size` to `GB` if it's set to `MB`.
-	- Set the `File System` to `FAT32`.
-	- Set the `Partition Label` to `SD`.
-	- Set the `Partition Size` to `8.00`.
-	- Set the `Unallocated Space After` to `29872`.
-	- Drag the left arrow of the partition completely to the left, filling all the space to the left.
-	- Set the `Unallocated Space Before` to `16`.
-	- If done correctly, the `Unallocated Space After` should now display as `29856` instead of `29872`.
-
-		![Create_SD_partition](../img/minitool1.png)
-
-
-
-	- The final `Partition Size` of the FAT32 partition will vary depending on your SD card's total size.
-
-5. Create another new partition with the following settings:
-	- Ignore the warning about Windows not being able to recognize the partition.
-	- Set the `File System` to `Unformatted`.
-	- If Minitool allows you to, set the `Drive Letter` to `None`. If the option is already disabled/greyed out, continue to the next step.
-	- Set the metrics on the right of `Unallocated Space Before`, `Partition Size`, and `Unallocated Space After` from `GB` to `MB`.
-	- `Unallocated Space Before` should display as `0MB`.
-	- `Partition Size` should display as `29856MB`.
-	- `Unallocated Space After` should display as `0MB`.
-
-
-		![Create_emummc_partition](../img/minitool2.png)
+## Preperations
 
+What you need:
+- The latest release of <a href="https://github.com/suchmememanyskill/TegraExplorer/releases" target="_blank">TegraExplorer</a>
 
-6. Your SD card now should look similar to this:
-   
-	![SD_format](../img/emummc_final_minitool.png)
+### Instructions
 
-7. Apply the changes to your SD card by hitting `Apply` in the top left.
+1. Inject the TegraExplorer Payload with your 64gb+ sd card inserted into your switch
+	- If you forgot how to do this, re-read the [sending payload](sending_payload.md) section of the guide.
+2. Navigate to `SD Format` and press the power button to enter the SD format menu
+3. Navigate to `Format for EmuMMC setup` and press the power button to confirm
+5. Read the warning, and press Power after 10 seconds to format your sd card
+	- Note: This will delete all data on your sd card. Make sure you backed up your nintendo folder!
+	- This will take about 5 minutes
+6. Press any button to return to the main menu
+7. Navigate to `Exit` and press the power button to enter the Exit menu
+8. Navigate to `Reboot to RCM` and press the power button to reboot to RCM. It's now safe to eject your sd card for the next part of the guide
 
 &nbsp;
 
 #### [Continue to SD Preparations <i class="fa fa-arrow-circle-right fa-lg"></i>](sd_preparation.md)
 
 -----
-## Linux Instructions
+## Manual Linux Preperations
```

Comment:
Just remove the linux instructions. If TegraExplorer can take care of this, it's not needed.

Reply:
Linux instructions are now on a completely separate page